samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/readme.md

```diff
@@ -0,0 +1,162 @@
+# Get started sending and receiving messages from ServiceBus queues using QueueClient
+
+In order to run the sample in this directory, replace the following bracketed values in the `Program.cs` file.
+
+```csharp
+const string ServiceBusConnectionString = "{ServiceBus connection string}";
+const string QueueName = "{Queue path/name}";
+```
+
+Once you replace the above values run the following from a command prompt:
+   
+```
+dotnet restore
```

Comment:
dotnet commands are not replaced by msbuild. cli commands were replatted to use msbuild underneath. The main microsoft dotnet core cli getting started page, https://www.microsoft.com/net/core#windowscmd still uses dotnet cli as we are using here. As such I'm inclined to leave them as is.